bzt/modules/blazemeter.py

```diff
@@ -1105,7 +1344,7 @@ def get_labels(self, master_id):
 
     def get_available_locations(self):
         user_info = self.get_user_info()
-        return {str(x['id']): x for x in user_info['locations'] if not x['id'].startswith('harbor-')}
```

Comment:
Will this mean that old-style tests will allow Docker locations and will fail to handle it? There should be some kind of logic to exclude them for old-style tests and include in new-style.